051-MicrosoftPurview/Student/Challenge7.md

```diff
@@ -0,0 +1,16 @@
+# Challenge 7: Data Insights
+
+[< Previous Challenge](./Challenge6.md) - [Home](../README.md) - [Next Challenge >](./Challenge8.md)
+
+## Introduction
+There is quite a lot of metadata available already in Fabrikam's data catalog. You have received an ask from Chief Data Officer in your organization to have a birds eye view of the catalog. The CDO is looking for information about what sources you have in the environment and the type of data etc.
+
+## Description
+With your current Microsoft Purview environment, you are able to see individual metadata about your assets, business vocabulary in a glossary and lifecycle of the data. There may be areas which require you to revisit and put some additional work. Things like monitoring the scans, checking which glossary terms are being used or maybe get better understanding of the sources you have scanned before. With Data Insights you can have a single pane of glass view into the catalog. This will give you better understanding of existing assets and will provide reports on each section. This information may be valid for you to understand actions which happened in your Microsoft Purview and also understand where there is more work required.
```

Comment:
Change to "There may be areas which require you to revisit and put some additional work such as monitoring the scans, checking which glossary terms are being used or maybe getting a better understanding of the sources you have scanned before."

Reply:
Done